pkg/config/config.go

```diff
@@ -27,18 +30,21 @@ func New() Config {
 		PostgresContainerName:   getEnv("POSTGRES_CONTAINER_NAME", "postgresql"),
 		OauthProxyContainerName: getEnv("OAUTH_PROXY_CONTAINER_NAME", "ups-oauth-proxy"),
 
-		UPSImageStreamName:        getEnv("UPS_IMAGE_STREAM_NAME", "ups-imagestream"),
-		UPSImageStreamTag:         getEnv("UPS_IMAGE_STREAM_TAG", "latest"),
+		UPSImageStreamName: getEnv("UPS_IMAGE_STREAM_NAME", "ups-imagestream"),
+		UPSImageStreamTag:  getEnv("UPS_IMAGE_STREAM_TAG", "latest"),
+
 		OauthProxyImageStreamName: getEnv("OAUTH_PROXY_IMAGE_STREAM_NAME", "ups-oauth-proxy-imagestream"),
 		OauthProxyImageStreamTag:  getEnv("OAUTH_PROXY_IMAGE_STREAM_TAG", "latest"),
 
 		PostgresImageStreamNamespace: getEnv("POSTGRES_IMAGE_STREAM_NAMESPACE", "openshift"),
 		PostgresImageStreamName:      getEnv("POSTGRES_IMAGE_STREAM_NAME", "postgresql"),
-		PostgresImageStreamTag:       getEnv("POSTGRES_IMAGE_STREAM_TAG", "latest"),
+		PostgresImageStreamTag:       getEnv("POSTGRES_IMAGE_STREAM_TAG", "9.6"),
 
 		// these are used when the image stream does not exist and created for the first time by the operator
 		UPSImageStreamInitialImage:        getEnv("UPS_IMAGE_STREAM_INITIAL_IMAGE", "docker.io/aerogear/unifiedpush-wildfly-plain:2.2.1.Final"),
 		OauthProxyImageStreamInitialImage: getEnv("OAUTH_PROXY_IMAGE_STREAM_INITIAL_IMAGE", "docker.io/openshift/oauth-proxy:v1.1.0"),
+
+		BackupImage: getEnv("BACKUP_IMAGE", "quay.io/integreatly/backup-container:master"),
```

Comment:
are we always going to use master? is there value in using it as a cr param?

Reply:
I'm not sure what tag we should use by default here, maybe a floating `1.0` tag that moves with patch releases to that stream would be better than master (which could break things here)? Having it at specific releases could fall out-of-date quickly.

I don't think it should be part of the CR though. It's configurable through the `BACKUP_IMAGE` env var to the operator here.

Reply:
I am asking because that image changes quite often from a release perspective and how easy would it be to use and maintain  a fixed number rather than use master which may be unstable

Reply:
Yeah, I think you're right. I'll change the default to a fixed tag :+1: